.github/workflows/deploy.yml

```
@@ -37,15 +37,16 @@ jobs:
           tar -xf iota.tar
           tar -xf shimmer.tar
           tar -xf next.tar
-      # Convert to Vercel project and upload
-      - name: Install Vercel CLI
-        run: npm install --global vercel@latest
-      - name: Pull Vercel Environment Information
-        run: vercel pull --yes --environment=production --token=${{ secrets.VERCEL_TOKEN }}
-      - name: Build Project Artifacts
-        run: vercel build --prod --token=${{ secrets.VERCEL_TOKEN }}
-      - name: Deploy Project Artifacts to Vercel
-        run: vercel deploy --prebuilt --prod --token=${{ secrets.VERCEL_TOKEN }}
+      # Deploy to Vercel
+      - name: Deploy to Vercel Action
+        uses: BetaHuhn/deploy-to-vercel-action@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          VERCEL_TOKEN: ${{ secrets.VERCEL_TOKEN }}
+          VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+          VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
+          PR_LABELS: false
```

I think it would be good to add `PRODUCTION: true` here just in case

---

Should be on by default, but we can be explicit 👍 
```suggestion
          PR_LABELS: false
          PRODUCTION: true
```